build/buildpipeline/windows.groovy

```diff
@@ -2,7 +2,7 @@
 
 // 'node' indicates to Jenkins that the enclosed block runs on a node that matches
 // the label 'windows-with-vs'
-simpleNode('Windows_NT','latest') {
+simpleNode('Windows.10.Enterprise.RS3.ASPNET') {
```

Comment:
I don't really know what this is.

Reply:
To fix the dotnetbot CI test failure. This fix is already present in dev.

Reply:
It's an newer image with VS 15.6 or later. The other one has super ancient VS installed.